### ISSUE TITLE
Update message format page on EGI wiki

### ISSUE THREAD

**Issue by tofu-rocketry** [apel/apel#212]
The page we usually point to for message structure is out of date and out of step with the actual message format: https://wiki.egi.eu/wiki/APEL/MessageFormat

**Comment by rptaylor** [comment]
@tofu-rocketry what is the current format?  v0.2, v0.3 ?
Is it different from this ? https://wiki.egi.eu/wiki/APEL/MessageFormatV02

I need the current format to test publishing k8s accounting data.
Thanks!

**Comment by rptaylor** [comment]
Any update on this @tofu-rocketry  ? It has become quite urgent.
Thanks!

**Comment by gregcorbett** [comment]
As pointed out in [Ticket 152402](https://ggus.eu/index.php?mode=ticket_info&ticket_id=152402), whilst EarliestEndTime and LatestEndTime aren't mandatory in the message format, they are used by the portal to assign usage to months/years.

We should consider:

- making those fields mandatory in the record format and for loading into the database,
- asking the portal to use the `Month` and `Year` fields to assign grid usage to months/years, as they do for Cloud accounting.

**Comment by rptaylor** [comment]
> asking the portal to use the `Month` and `Year` fields to assign grid usage to months/years, as they do for Cloud accounting.

Makes sense to me, the Month and Year are already provided so this could allow EarliestEndTime and LatestEndTime to be omitted since they are redundant.

**Comment by rptaylor** [comment]
Can we have documentation of the required parameters for publishing records with HS23 benchmark values? 

It looks like the documentation has not been updated since 2014, also there is a banner warning that wiki was to be decommissioned a year ago:
https://wiki.egi.eu/wiki/APEL/MessageFormat

**Comment by tofu-rocketry** [comment]
> Can we have documentation of the required parameters for publishing records with HS23 benchmark values?
> 
> It looks like the documentation has not been updated since 2014, also there is a banner warning that wiki was to be decommissioned a year ago: https://wiki.egi.eu/wiki/APEL/MessageFormat

For HEPscore23, we've been keeping [details on the CERN TWiki](https://twiki.cern.ch/twiki/bin/view/LCG/ChangesForHEPscore) for now. There is also a link to the spec document on that page.

**Comment by rptaylor** [comment]
Thanks @tofu-rocketry , can you clarify if the proposed specification in the [google doc](https://docs.google.com/document/d/19EerfwmzhUM4gLijOK9XxAM46vg1E2oSJvWVoEltvRA/edit?pli=1#heading=h.vlqqhvvfi4e4) has been implemented?  Are the production APEL servers ready to receive the new v0.4 format of records or is this still being tested, and if the latter can I send records to the test server?
